### ISSUE TITLE
Spells/ Cantrips tracker 

### ISSUE THREAD

**Issue by eehresmann94** [bettet01/dnd_Char_creator#10]
Create a method/ sheet that can track the cantrips and spells of each character as well as create a way to print out a quick summary of their effects to the cmd line for easy reference.  

Starting points: 
1) create a char_spell.csv 
2) import the name, class,and race from the master.csv to ensure accuracy. 
3) create columns for cantrips, and different level spells. 
    a) if possible create a way to the number of spell slots that each character has based upon their level. 
4) Provide a summary of a given spells taken from beyond dnd for each class. 
     - https://www.dndbeyond.com/spells 

! This might be a good chance to experiment with embedding a database into our program since there are many spells for each class and a csv may be too limited with respect to information that it can hold.

**Comment by bettet01** [comment]
i agree i think at some point we should have a database of all the spells for each class, but this time i'm thinking that we can have levels and spells as a dict class and push the dictionary to the character csv file.
